mne_bids/commands/mne_bids_raw_to_bids.py

```diff
@@ -30,7 +30,7 @@ def run():
     parser.add_option(
         "--subject_id",
         dest="subject_id",
-        help=("subject name in BIDS compatible format " "(01, 02, etc.)"),
+        help=("subject name in BIDS compatible format (01, 02, etc.)"),
```

Comment:
I don't think the parentheses are needed here

Reply:
I'm surprised `ruff` does not fix those automatically :/